packages/kbn-apm-synthtrace-client/src/lib/otel/error.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import type { OtelDocument } from '../../..';
+import { Serializable } from '../serializable';
+
+export interface OtelErrorDocument extends OtelDocument {
+  'event.name'?: string;
+  attributes?: {
+    'exception.message'?: string;
+    'error.stack_trace'?: string;
```

Comment:
I think we can add '`exception.handled'` as per this comment https://github.com/elastic/kibana/issues/194123#issuecomment-2378646022

Reply:
Thanks for the comment @MiriamAparicio! 
@gregkalapos Is the 'exception.handled' present in Otel now? I remember there were some discussions that it is optional or hard to add but I am not aware of the latest changes.

Reply:
Yes, it's there now.

Reply:
Added, thanks for confirming 👍